config/checkstyle_input_suppressions.xml

```diff
@@ -892,6 +890,22 @@
             files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocUnclosedTagAndInvalidAtSeeReference.java"/>
   <suppress id="ConfigCommentOnTopInputs"
             files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocWrongSingletonTagInJavadoc.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocInvalidAtSeeReference2.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocLeaveToken.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocNonTightHtmlTagsNoViolation.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocNonTightHtmlTagsVisitCount.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocParsingErrors2.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensFail.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensPass.java"/>
+  <suppress id="ConfigCommentOnTopInputs"
+            files="javadoc[\\/]abstractjavadoc[\\/]InputAbstractJavadocTokensPass.java"/>
```

Comment:
Please do not add suppression. Add a comment like below at top of the file (just above class), so as violations are not reported.
```
/**
 * Config:
 * exampleSetting = exampleValue
 */
```

Reply:
@AkMo3 Of course, I know that. It was too late in the night yesterday so I simply created a PR (instead of a draft one by mistake). I'll push changes in a while. Thanks!